client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-button.jsx

```diff
@@ -48,6 +49,9 @@ export default React.createClass( {
 			scary: true,
 			onClick: ( event ) => {
 				event.preventDefault();
+				if ( this.props.isMock ) {
+					return;
```

Comment:
I think the idea is that we still want the toggles and the disconnect buttons visible. Since they show the possible actions when that would be available to the user.